apps/web/app/api/partner-profile/programs/[programId]/earnings/timeseries/route.ts

```diff
@@ -5,9 +5,8 @@ import { sqlGranularityMap } from "@/lib/planetscale/granularity";
 import { analyticsQuerySchema } from "@/lib/zod/schemas/analytics";
 import { prisma } from "@dub/prisma";
 import { Prisma } from "@prisma/client";
-import { format } from "date-fns";
+import { DateTime } from "luxon";
```

Comment:
Ended up adding Luxon for proper timezone handling.

I tried using `date-fns` with `date-fns-tz` to convert to/from the specified timezone, but this couldn't handle DST properly because the functions used in `dateIncrement` still operate in the server timezone (UTC)

Reply:
nice! we should take some time soon to standardize all these across our codebase btw 🙏